### ISSUE TITLE
PhoSim Instrumental Configuration does not run

### ISSUE THREAD

**Issue by danielsf** [LSSTDESC/DC2_Repo#231]
It cannot find "segmentation.txt"

@cwwalter @slosar

**Comment by cwwalter** [comment]
So, I wrote this notebook as a utility for people who are doing work on production. The idea was if you had a segmentation.txt file from some version of PhoSim, you could use it to explore what was inside to extract x-talk values etc.  

Later, the tutorial notebooks were added to the Notebook directory along with the auto-run output etc. 

I was going to suggest we make a subdirectory structure for tutorials, utilities etc and separate them.  But in the discussion in #238 by @yymao and @jchiang87 etc it looks like we will be splitting into at least a DC2 production and analysis repo.  I suggest this one stay here and not be part of an auto build system.  In the mean time, I can also add some text to the cell pointing out you should use your own segmentation.txt file.  I'm wary of adding code to automatically fetch one, or check one in, since people might look at different versions than they really need.

**Comment by yymao** [comment]
Yes, I agree that if we split into `DC2-production` and `DC2-analysis` then Chris' notebook should stay in `DC2-production`, and that would reduce the potential confusion that a general DC2 user might have.
